cfgov/unprocessed/js/modules/util/dom-events.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+/**
+ * Parses the event passed by the bind shortcut to create the event listener
+ * @param   {string}   evt      The type of event to watch for
+ * @param   {HTMLNode} elem     The element to watch for the passed event
+ * @param   {Function} callback The function to trigger on the event
+ */
+function _parseEvent( elem, evt, callback ) {
+  evt.split( /\s+/ ).forEach(
+    function( event ) {
+      elem.addEventListener( event, callback );
+    }
+  );
+}
+
+/**
+ * Shortcut for binding event listeners to elements
+ * @param  {HTMLNode} elem    The element to attach the event listener to
+ * @param  {Object}   options The options for the event listener
+ */
+function bindEvent( elem, options ) {
+  var callback;
+  for ( var evt in options ) {
+    if ( options.hasOwnProperty( evt ) ) {
+      callback = options[evt];
+      _parseEvent( elem, evt, callback );
+    }
+  }
+}
+
+/**
+ * Shortcut for triggering events on elements
+ * @param  {HTMLNode} target     The element to fire events on
+ * @param  {string}   type       The event type to create
+ * @param  {Object}   properties The options for the event
+ */
+function fireEvent( elem, type, properties ) {
+  var evt = document.createEvent( 'HTMLEvents' );
+
+  evt.initEvent( type, true, true );
+
+  for ( var j in properties ) {
+    if ( properties.hasOwnProperty( j ) ) {
+      evt[j] = properties[j];
+    }
+  }
+
+  elem.dispatchEvent( evt );
```

Comment:
At some point we'll want to maybe investigate overlaps between this and https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/unprocessed/js/modules/util/EventObserver.js#L58 ... though I see how they fill different roles, one for one-off dispatching and the other for creating a `dispatchEvent` method on an instance.